engine/dlib/src/dlib/buffer.h

```diff
@@ -74,6 +74,10 @@ Result GetStreamOffset(HBuffer buffer, uint32_t index, uint32_t* offset);
 
 Result CalcStructSize(uint32_t num_streams, const StreamDeclaration* streams, uint32_t* size, uint32_t* offsets);
 
+Result SetMetaData(HBuffer hbuffer, dmhash_t name_hash, const void* data, uint32_t count, ValueType type);
+
+Result GetMetaData(HBuffer hbuffer, dmhash_t name_hash, void** data, uint32_t* count, ValueType* type);
```

Comment:
I see no reason why these shouldn't be in the public api header (with documentation)?

Reply:
No reason at all. I just didn't know whether they were supposed to be public or not. 

Moving them to dmsdk/dlib/buffer.h.